src/core/event/EventManager.js

```diff
@@ -0,0 +1,60 @@
+/* global console */
```

Comment:
Filenames. Convention in luma.gl is to use `underscore_case` for filenames, matching names of classes that are in `PascalCase`.

So this would be `event-manager.js`